pkgs/applications/video/mpv/scripts/buildLua.nix

```diff
@@ -75,9 +76,20 @@ lib.makeOverridable (
           runHook postInstall
         '';
 
-        passthru = {
-          inherit scriptName;
-        };
+        passthru =
+          {
+            inherit scriptName;
+          }
+          // lib.optionalAttrs (runtime-dependencies != [ ]) {
+            extraWrapperArgs =
+              [
+                "--prefix"
+                "PATH"
+                ":"
+              ]
+              ++ (map lib.makeBinPath runtime-dependencies)
```

Comment:
```suggestion
              ++ (lib.makeBinPath runtime-dependencies)
```

Reply:
Yes sorry, I had noticed and fixed that locally, but failed to push it to various branches